### DISCUSSION TITLE
Providing an edge-case option to back up without "compressing"?

### DISCUSSION THREAD

**Discussion by BenClementt** [AzuraCast/AzuraCast#5945] [Ideas]
Hey there, aiming this one at Buster if he has time to respond:

I feel like it could be a good idea (maybe only for me :P) to add a flag for being able to run the docker backup script without actually compressing into the tar file, only storing?

Similar to zip -0.

Feel like it could be handy in situations where network speed is not an issue, but time is. 
For reference, I have an AzuraCast instance with 30+ stations and over 300gb of media and metadata.

I do know this is a pretty niche thing though, so I do understand the possible hesitation in adding it.

Thanks for providing such great software, Buster <3

**Comment by BusterNeece** [comment]
@BenClementt If I recall correctly, with our `zip` backup format, we expressly specify all of the music file formats as uncompressable so they just get added as-is, so it's quite a lot faster. There's so little benefit to compressing those files that we prefer not to if we can.

**Reply by BenClementt** [reply]
Using the default CLI backup script, it does seem to compress the mp3 files, unless I'm totally missing something - Usually just storing files in a zip does run a lot faster than what I can see from the output.

Not sure if I'm just missing something here.

**Comment by LordHelmchen666** [comment]
I use rsync for this case.
My  docker-compose.override.yml
```
services:
 web:
  volumes:
      - /private-backup/azuracast:/var/azuracast/backups
      - /home/azuracast/stations:/var/azuracast/stations
```
To backup on same server to another folder (cronjob on Azuracast-Server once a day)
`rsync -avh --delete /home/ /private-backup/home/`

To copy backups on Backup-Server (cronjob on Backup-Server  once a day)
`rsync -avzhe ssh --delete root@AZURACAST-IP:/private-backup/azuracast/ /home/backups/azuracast_zip/`

To copy stations (config, media, albumart, etc.) on Backup-Server (cronjob on Backup-Server  once a day)
`rsync -avzhe ssh --delete root@AZURACAST-IP:/home/azuracast/stations/ /home/backups/azuracast_stations/`

**Comment by BenClementt** [comment]
This is very similar to what I ended up doing tbh, good shout!

**Comment by LordHelmchen666** [comment]
And i use zstd (ZStandard) for backups. It is faster than zip for me. https://github.com/facebook/zstd

**Reply by BenClementt** [reply]
@BusterNeece Looks pretty nice to be honest - Not sure how well it would work in the specific AzuraCast instance. @LordHelmchen666 - Has it been faster for you adding mp3 files and other supplementary data?

**Reply by LordHelmchen666** [reply]
Yes, it is faster with mp3 and other data for me. There are a lot of benchmarks out there. Just look at them. ;)

**Reply by BusterNeece** [reply]
@BenClementt We do support ZStd compression for backups already in our installation.

**Reply by BenClementt** [reply]
@BusterNeece Am I missing a trick with some sort of CLI flag to select the compression method?

**Reply by BusterNeece** [reply]
@BenClementt It's the extension of the file, so `.tzst` is the format we use to select ZStd.
